src/app/presentation/components/add-workspace/add-workspace.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { WorkspaceService } from 'src/app/core/services/workspace.service';
+import { Router } from '@angular/router';
+import { CreateWorkspace } from 'src/app/core/models/workspace';
+import Swal from 'sweetalert2';
+
+
+@Component({
+  selector: 'app-add-workspace',
+  templateUrl: './add-workspace.component.html',
+  styleUrls: ['./add-workspace.component.css']
+})
+
+export class AddWorkspaceComponent implements OnInit {
+  addwsform : FormGroup;
+  createWorkspace : CreateWorkspace = new CreateWorkspace();
+  
+
+   
+  constructor(private fb:FormBuilder, private service: WorkspaceService, private router: Router) { 
+    this.addwsform=fb.group({        
+      "Name": ["",[Validators.required]],
+      "Description" : ["",[Validators.required]]               
```

Comment:
Why first parameters are empty ?

Reply:
Becouse, if I write for example ["test",[Validators.required]],then after start page we have this "test" just in the field, and when we want to write our name we have to delete this "test" and after that we write our name. But if we have ["",[Validators.required]] we at once write our name. I think it wiil more simple for user